test/Conversion/TorchToTosa/basic.mlir

```diff
@@ -977,3 +977,29 @@ func.func @torch.aten.where.self(%arg0: !torch.vtensor<[1,1,5,5],i1>, %arg1: !to
   %0 = torch.aten.where.self %arg0, %arg1, %arg2 : !torch.vtensor<[1,1,5,5],i1>, !torch.vtensor<[1,12,5,5],f32>, !torch.vtensor<[],f32> -> !torch.vtensor<[1,12,5,5],f32>
   return %0 : !torch.vtensor<[1,12,5,5],f32>
 }
+
+// -----
+// -----
+// CHECK-LABEL:   func.func @torch.aten.hardtanh_backward(
+// CHECK-SAME:                       %[[VAL_0:.*]]: !torch.vtensor<[10,20],f32>,
+// CHECK-SAME:                       %[[VAL_1:.*]]: !torch.vtensor<[10,20],f32>) -> !torch.vtensor<[10,20],f32> {
+// CHECK:           %[[VAL_2:.*]] = torch_c.to_builtin_tensor %[[VAL_0]] : !torch.vtensor<[10,20],f32> -> tensor<10x20xf32>
+// CHECK:           %[[VAL_3:.*]] = torch_c.to_builtin_tensor %[[VAL_1]] : !torch.vtensor<[10,20],f32> -> tensor<10x20xf32>
+// CHECK:           %[[VAL_4:.*]] = torch.constant.float 5.000000e-01
+// CHECK:           %[[VAL_5:.*]] = torch.constant.float 2.000000e-01
+// CHECK:           %[[VAL_6:.*]] = "tosa.const"() {value = dense<2.000000e-01> : tensor<1x1xf32>} : () -> tensor<1x1xf32>
+// CHECK:           %[[VAL_7:.*]] = "tosa.const"() {value = dense<5.000000e-01> : tensor<1x1xf32>} : () -> tensor<1x1xf32>
+// CHECK:           %[[VAL_8:.*]] = "tosa.const"() {value = dense<0.000000e+00> : tensor<f32>} : () -> tensor<f32>
+// CHECK:           %[[VAL_9:.*]] = "tosa.greater"(%[[VAL_6]], %[[VAL_3]]) : (tensor<1x1xf32>, tensor<10x20xf32>) -> tensor<10x20xi1>
+// CHECK:           %[[VAL_10:.*]] = "tosa.greater"(%[[VAL_3]], %[[VAL_7]]) : (tensor<10x20xf32>, tensor<1x1xf32>) -> tensor<10x20xi1>
+// CHECK:           %[[VAL_11:.*]] = "tosa.logical_or"(%[[VAL_9]], %[[VAL_10]]) : (tensor<10x20xi1>, tensor<10x20xi1>) -> tensor<10x20xi1>
+// CHECK:           %[[VAL_12:.*]] = "tosa.select"(%[[VAL_11]], %[[VAL_8]], %[[VAL_2]]) : (tensor<10x20xi1>, tensor<f32>, tensor<10x20xf32>) -> tensor<10x20xf32>
+// CHECK:           %[[VAL_13:.*]] = torch_c.from_builtin_tensor %[[VAL_12]] : tensor<10x20xf32> -> !torch.vtensor<[10,20],f32>
+// CHECK:           return %[[VAL_13]] : !torch.vtensor<[10,20],f32>
+
+func.func @torch.aten.hardtanh_backward(%arg0: !torch.vtensor<[10,20],f32>, %arg1: !torch.vtensor<[10,20],f32>) -> !torch.vtensor<[10,20],f32> {
```

Comment:
This is no longer needed, since it's just testing a macro expansion. See: https://github.com/llvm/torch-mlir/blob/main/docs/architecture.md#dos-and-donts-for-unit-vs-end-to-end-testing

> DON'T use a unit test if your lowering pattern could be described as a trivial "macro expansion" of one op into another op or set of ops. That is, if you feel like your unit test is just rewriting b.create<...>(...) into CHECK: ... then it is probably not a useful unit test.